### ISSUE TITLE
[Rule Assistant] value attribute gets filled in instead of match

### ISSUE THREAD

**Issue by rmlockwood** [rmlockwood/FLExTrans#653]
I built the following rule using the new 'unmarked' capability. See image below.

It looks like the UI is writing the attributes wrong for this suffix. It is coming out as:

```
> <Affixes>
> 	<Affix type="suffix">
> 		<Features>
> 			<Feature label="number" match="" ranking="0" unmarked_default="sg" value="β"/>
> 		</Features>
> 	</Affix>
> </Affixes>

```
It should have match="β" not value="β"

![image](https://github.com/rmlockwood/FLExTrans/assets/7088137/054a7ac8-13c0-4942-945c-62bb48110974)

**Comment by AndyBlack** [comment]
The fix should be in version 0.15.1 of the UI at https://drive.google.com/file/d/18wkEePZGmBqxdhYYBByIsgGZjCDTAJBu/view?usp=sharing

**Comment by rmlockwood** [comment]
This works for me in 0.16
